etna/transforms/math/binary_operator.py

```diff
@@ -0,0 +1,124 @@
+from enum import Enum
+from typing import List
+from typing import Optional
+
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.transforms.base import ReversibleTransform
+
+
+class MathOperator(str, Enum):
```

Comment:
I think Its better to create two separate classes: `ArithmeticOperator` and `ComparisonOperator`

Reply:
You should add `_missing_` method